README.md

```diff
@@ -76,6 +76,17 @@ MyType.prototype[fl.map] = MyType.prototype.map
 var foo = bar[fl.map](x => x + 1)
 ```
 
+## Type representatives
+
+Certain behaviours are defined from the perspective of a member of a type.
+Other behaviours do not require a member. Thus certain algebras require a
+type to provide a value-level representative (with certain properties). The
+Identity type, for example, could provide `Id` as its type representative:
+`Id :: TypeRep Identity`.
```

Comment:
In Haskell at least those last equations are wrong. 

- Just constructs values at the Maybe type, it's not a representative of the type. In particular, a type should be represented in the shape of the type, not its values.
- Maybe is an interesting example since it's a type constructor, a type function. If we want to represent this is a value-level type representative then `TypeRep Maybe` and `TypeRep X` must be able to be combined into `TypeRep (Maybe X)`. Perhaps the type reps of constructors are functions, or probably better (since it'll be easier to analyze) you could have something that gives you `($$) :: TypeRep f -> TypeRep a -> TypeRep (f a)`